feature-libs/pickup-in-store/core/facade/cart.service.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { TestBed } from '@angular/core/testing';
+import { CartService } from './cart.service';
+
+describe('CartService', () => {
+  let service: CartService;
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [CartService],
+    });
+    service = TestBed.inject(CartService);
+  });
+
+  it('should call getPickupOption and return object', () => {
+    const received = service.getPickupOption('productCode');
+
+    expect(received).toBeUndefined();
+  });
+});
```

Comment:
What is the purpose of this test? It asserts that a void function returns nothing?